lib/index.ts

```diff
@@ -183,7 +183,7 @@ export class Front {
 
 		// Listen for Webhooks on the path specified by the client.
 		listener.post(hookPath, (req: express.Request, res: express.Response) => {
-			const eventPreview: EventPreview = req.body;
+			const eventPreview: EventPreview = typeof(req.body) === 'string' ? JSON.parse(req.body) : req.body;
```

Comment:
This changes the documentation, namely:
```
You can also use a pre-existing Express instance, but note that the Front SDK expects that you have request body parsing of JSON enabled (as it will not do this for you). You can easily add this using the body-parser library:
```
So either this documentation either needs changing, or the user should honour it by passing in an express instance that conforms to it. Is there a particular use case you have where this is not true?

Reply:
I assume it's because of this: https://github.com/resin-io-modules/resin-procbots/pull/309#issuecomment-345702236

So we can do this, but you need to change the documentation and please can we add parentheses around the ternary comparator so the assignment and test are clearly delineated.